src/Eccube/Common/EccubeConfig.php

```diff
@@ -85,7 +76,7 @@ public function offsetGet($offset)
     #[\ReturnTypeWillChange]
     public function offsetSet($offset, $value)
     {
-        $this->set($offset, $value);
+        throw new \Exception();
```

Comment:
これが呼ばれるのはプログラミングのミスだと思うので、 LogickException で良いと思います
```suggestion
        throw new \LogicException();
```

Reply:
ありがとうございます、修正します。